modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-kubernetes-configuring_{context}"]
+= Configuring the TLS security profile for the control plane
+
+To configure a TLS security profile for the control plane, edit the `APIServer` custom resource (CR) to specify a predefined or custom TLS security profile. Setting the TLS security profile in the `APIServer` CR propagates the setting to the following control plane components:
+
+* Kubernetes API server
+* OpenShift API server
+* OpenShift OAuth API server
+* OpenShift OAuth server
+
+If a TLS security profile is not configured, the default TLS security profile is `Intermediate`.
+
+[NOTE]
+====
+The default TLS security profile for the Ingress Controller is based on the TLS security profile set for the API server.
+====
+
+.Sample `APIServer` CR that configures the `Old` TLS security profile
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+ ...
+spec:
+  tlsSecurityProfile:
+    old: {}
+    type: Old
+ ...
+----
+
+The TLS security profile defines the minimum TLS version and the TLS ciphers required to communicate with the control plane components.
+
+You can see the configured TLS security profile in the `APIServer` custom resource (CR) under `Spec.Tls Security Profile`. For the `Custom` TLS security profile, the specific ciphers and minimum TLS version are listed.
```

Comment:
The word `Spec.Tls Security Profile` , it should be under `Spec.tlsSecurityProfile` in CR.

Reply:
@mburke5678 did you display it as `Spec.Tls Security Profile` because that's how it appears in the verification steps when you use `oc describe`?

I agree this is a little confusing, because we haven't actually set it yet (since this is before the procedure) and we do usually write the fields how they look in the YAML. Thoughts on making an update?

@snarayan-redhat I'd say leave this part alone for now, and if we make an adjustment, we'll fix it for all versions